src/pages/active.tsx

```diff
@@ -1,87 +1,69 @@
-import { FC, useState, useEffect } from 'react';
-import Head from '@/components/head';
-import Layout from '@/components/Layout';
-import Card from '@/components/tasks/card';
-import useFetch from '@/hooks/useFetch';
-import classNames from '@/styles/tasks.module.scss';
-import task from '@/interfaces/task.type';
+import { FC, useState, useEffect } from "react";
+import Head from "@/components/head";
+import Layout from "@/components/Layout";
+import Card from "@/components/tasks/card";
+import useFetch from "@/hooks/useFetch";
+import classNames from "@/styles/tasks.module.scss";
+import task from "@/interfaces/task.type";
 import {
-  BLOCKED, IN_PROGRESS, SMOKE_TESTING,
-} from '@/components/constants/task-status';
-import updateTasksStatus from '@/helperFunctions/updateTasksStatus';
-import beautifyTaskStatus from '@/helperFunctions/beautifyTaskStatus';
+  BLOCKED,
+  IN_PROGRESS,
+  SMOKE_TESTING,
+} from "@/components/constants/task-status";
+import updateTasksStatus from "@/helperFunctions/updateTasksStatus";
+import beautifyTaskStatus from "@/helperFunctions/beautifyTaskStatus";
 
-const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
+const MY_TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
```

Comment:
https://github.com/Real-Dev-Squad/website-status/pull/341/commits/1646616b551c9509d8519d80b4ed76298cdf6e0c#r1122712283

why the variable name is different form above

Reply:
In oder to make it more meaningful and to differentiate TASKS_URL & TASK_URL
I replaced it in such a way.